system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -349,3 +377,23 @@
       (index/wait-until-indexed)
       (let [collection-response (ingest/bulk-update-task-status "PROV1" (:task-id response))]
         (is (= "COMPLETE" (:task-status collection-response)))))))
+
+(deftest bulk-update-large-status-message-test
+  (let [concept-ids (ingest-collection-in-umm-json-format large-status-message-umm) 
+        _ (index/wait-until-indexed)
+        bulk-update-body {:concept-ids concept-ids
+                          :name "TEST NAME"
+                          :update-type "ADD_TO_EXISTING"
+                          :update-field "SCIENCE_KEYWORDS"
+                          :update-value {:Category "EARTH SCIENCE"
+                                         :Topic "HUMAN DIMENSIONS"
+                                         :Term "ENVIRONMENTAL IMPACTS"
+                                         :VariableLevel1 "HEAVY METALS CONCENTRATION"}}
+        task-id (:task-id (ingest/bulk-update-collections "PROV1" bulk-update-body))]
+    (index/wait-until-indexed)
+    (let [collection-response (ingest/bulk-update-task-status "PROV1" task-id)
+          collection-status (first (:collection-statuses collection-response))]
+      (is (= "COMPLETE" (:task-status collection-response)))
+      (is (= "COMPLETE" (:status collection-status)))
+      (is (< 255 (count (:status-message collection-status)))))))
+
```

Comment:
I may be missing some code/context here that already exists elsewhere ... I would think you'd need to test a message of under 4000 chars, 4000 char exactly, and over 4000 chars ...

Reply:
I'm not sure, 4000 is just as arbitrary as 255.  Its not like the number 4000 is tied to a specific requirement.  MMT just doesn't want their messages to be truncated.  It was truncated before at 255.

Reply:
4000 is the limit for VARCHAR2 sizes in Oracle 11g and earlier and Oracle 12c when not running in extended mode. I forgot to ask the MMT guys, but seeing that it is tied to a single collection, truncating at 4000 characters seems fine.

Note that with VARCHAR types we will only allocate as many bytes as are actually used for each row so oversizing the column is not a problem.

Reply:
Also I'm ok with this system integration test since we don't know of a way to generate a condition that would generate more than 4000 characters and we are at least verifying that we are not truncating at 255 anymore.

Ideally we'd have a test showing we truncate in a way that the data is successfully saved in Oracle even with special characters that take more than 1 byte in UTF-8 (like π). Is it possible to do this as a "unit with database test" where we manually call an appropriate function which sets the status-message to something like 5000 π characters?